### ISSUE TITLE
Bulk add: rotation is incorrectly filtering

### ISSUE THREAD

**Issue by city41** [city41/AMMiSTer#33]
Seems rotation in bulk add is totally broken in 0.18.0

@Eduardolicious said
```
 in .18 and still having issues with the new database and incorrectly selecting the filters. i.e. ROTATION = HORIZONTAL still giving tons of vertical results like arkanoid, space invaders, etc, while still also getting some horizontal titles like street fighter etc

changing to ROTATION != VERTICAL giving seemingly different results as well
```

**Comment by Eduardolicious** [comment]
playing with it some more, i think it might be the filters in general, not just rotation. attaching a screenshot, specialcontrols = spinner is giving all kinds of weird results, none of them being spinner games haha

![image](https://user-images.githubusercontent.com/67165184/213600279-a50b2e5a-c63e-4162-bd96-6bb3c2760d24.png)

**Comment by city41** [comment]
I just released 0.18.1 which fixes these. I had already fixed the special controls bug but hadn't cut a release yet. Thanks for finding these!
